relay-server/src/metrics_extraction/snapshots/relay_server__metrics_extraction__event__tests__extract_span_metrics_all_modules.snap

```diff
@@ -1308,6 +1308,24 @@ expression: metrics
             "transaction": "gEt /api/:version/users/",
         },
     },
+    Bucket {
+        timestamp: UnixTimestamp(1694732408),
+        width: 0,
+        name: "d:spans/http.decoded_response_body_length@byte",
+        value: Distribution(
+            [
+                128950.0,
+            ],
+        ),
+        tags: {
+            "environment": "fake_environment",
+            "resource.render_blocking_status": "blocking",
+            "span.description": "https://cdn.domain.com/path/to/file-hk2YHeW7Eo2XLCi*z*KuljsgCAD6hyWCyOYZM.css",
```

Comment:
Not relevant for this PR -- could this file get some improvements on normalization?

Reply:
Yes, we're considering a condition like "is long word with at least one numeric digit".